cpu/atmega_common/avr-libc-extra/unistd.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#ifndef UNISTD_H_
+#define UNISTD_H_
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define STDOUT_FILENO (0)
```

Comment:
Probably not really needed, but the old (correct) definition won't hurt, will it?
```c
#define STDIN_FILENO    0   ///< stdin file descriptor
#define STDOUT_FILENO   1   ///< stdout file descriptor
#define STDERR_FILENO   2   ///< stderr file descriptor
```